### PR TITLE
Add Usage to the readme

### PR DESCRIPTION
**Pull request by haikyuu**
Brings #3 solution and explanation into the Readme.

### CODE REVIEW

**Comment by haikyuu** [comment]
One question though, what's the use of `.esdl` as opposed to `.edgeql` files?

**Comment by 1st1** [comment]
`.esdl` files will contain schema definitions in [SDL](https://edgedb.com/docs/edgeql/sdl/index). They will become especially useful in alpha 3 when we add CLI tools to work with them.

File: README.md [hunk 1]
```diff
@@ -3,3 +3,10 @@
 This is a package with syntax highlighter for EdgeDB languages: EdgeQL and
 ESDL (EdgeDB Schema Definition Language).  The plugin is designed to work with
 vim and neovim.
+
+## Usage
+Copy this repo in your vim pack directory. More info in [Vim docs](https://github.com/vim/vim/blob/247bf0de465411e4ebb1c0fe1a9e07f4a7f77e91/runtime/doc/repeat.txt#L515)
```
Comment (1st1) [inline README.md:10]: There are actually two ways of installing edgedb-vim:

1. Use https://github.com/VundleVim/Vundle.vim or https://github.com/junegunn/vim-plug

2. Manual clone

Comment (haikyuu) [reply README.md:10]: @1st1 I added both now.

**Review by 1st1 — APPROVED** [review]

**Comment by 1st1** [comment]
Thank you!

**Pull request merged**